tools/c7n_azure/c7n_azure/functionapp_utils.py

```diff
@@ -22,6 +22,8 @@
 
 from c7n.utils import local_session
 
+from azure.mgmt.web.models import AppServicePlan, SkuDescription
+
 
 class FunctionAppUtilities(object):
```

Comment:
I wonder if now that we will support both dedicated and consumption function apps if we need to start naming things accordingly so it's clear this is the dedicated infrastructure.